### DISCUSSION TITLE
Advanced HTML CSS: Adding a Project to Animations section

### DISCUSSION THREAD

**Discussion by abirbhattacharya82** [TheOdinProject/curriculum#26390] [General]
### Describe your suggestion

The Animations section under Advanced HTML CSS is pretty theoritical and with very less scope in practical usage. It would be good for learners if there was a project included in this section like most of the other sections. <br>
I would like to suggest a simple project like a carousel of images would be nice implementations of css animations.

### Path

Full Stack Javascript

### Lesson Url

https://www.theodinproject.com/paths/full-stack-javascript/courses/advanced-html-and-css

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### (Optional) Additional Comments

I would like to happily work on this issue by providing with the project and necessary documents.

**Comment by 01zulfi** [comment]
An animated carousal of images project sounds nice. @TheOdinProject/intermediate-html-css what do y'all think?

**Comment by ManonLef** [comment]
I do like the idea, but there’s already an image carousel in [this lesson](https://www.theodinproject.com/lessons/node-path-javascript-dynamic-user-interface-interactions), that’s prefaced with 

> Animations are typically handled by CSS which is a little out of the scope of this lesson, but interactive stuff like this is no fun without a little motion! If you want to take a break and learn more about making stuff move [go watch this video](https://www.youtube.com/watch?v=8kK-cA99SA0).

Perhaps you have a suggestion on how these two exercises would not interfere with each other or become too repetitive?

**Reply by wise-king-sullyman** [reply]
What if we had a mini project that just added animation to the carousel built in that lesson, and explicitly discourage users from using animations in that lesson as is being discussed in #27247

**Comment by abirbhattacharya82** [comment]
Hi @ManonLef @01zulfi 
I have a better idea as well and I think this project is not yet done. <br>
I am suggesting making an analog clock using html, css. This project will have the applications of transition, transform as well as keyframes. So this will be like combination of all the topics taught in animation section.

**Comment by codyloyd** [comment]
I'm not opposed to having a project here, but I'm not a fan of any of the suggestions haha.

One project I've done in the past in a workshop was a little CSS/HTML robot.  We can provide images of it's body, head, legs, arms and learners would manipulate those to make the robot dance and wave.

... we could make it a little Odin lol

**Comment by 01zulfi** [comment]
^ that would be fun

**Comment by wise-king-sullyman** [comment]
Another proposed project idea for this section: https://github.com/TheOdinProject/curriculum/issues/27396

**Comment by thatblindgeye** [comment]
A project would definitely be a good addition. There's several ideas tossed in the hat currently, and we should try to decide on one to go with (unless others feel a project absolutely isn't necessary, which we can also have that discussion).

- An analog clock
- An image carousel (we could just yank the image slider from the Dynamic UI lesson out)
- A solar system
- A funky lil guy (Odin-bot)

@codyloyd @wise-king-sullyman @01zulfi what do y'all think? The easiest might be the image carousel since we already have some instruction on that. The solar system that is linked from #27396 above looks pretty cool, though has a lot of moving parts (maybe it isn't an issue considering where the project would be placed in the curriculum). The dancing Odin robot sounds pretty fun and I just like the idea of a dancing Odin-bot.

**Reply by wise-king-sullyman** [reply]
I'm also quite tempted by the dancing Odin-bot

**Reply by fadingmorsecode** [reply]
Just to chime in and say I immediately noticed the lack of a project here. Granted I'm not to the end of the course yet but it seems like a bit of info that would benefit from being reinforced by a small project. I like the idea of the Odin bot.

**Comment by thatblindgeye** [comment]
Closing this discussion in favor of an issue to move forward with a project addition.
